### ISSUE TITLE
Thank you, I like! Two little suggestions...

### ISSUE THREAD

**Issue by denislutz** [olefjaerestad/vscode-circular-dependencies-finder#7]
Hi thanks for this extension, it really find cyclic deps! Very useful for TS issues. Its a wonder why it has not so many stars but I am sure it will come....

My two little suggestions:

1. In the json view I can not search for strings to find the problematic classes by name
2. In the tree view, please increase the class names, you can rather, but this is a nice to have...

**Comment by olefjaerestad** [comment]
Glad you've found it helpful! 

1. I'm not sure I understand. Are you thinking of a freetext search? Something like a "cmd+f" that would highlight the matches? I'd be happy to help, I just need to understand the use case a bit better.
2. By increasing, I assume you're talking about the font size?

**Comment by denislutz** [comment]
Hi Olef,

I apologies for the late reply.

1. Yes, free text search, I want to search for suspicious file names if its in the set of cycles
2. Yes, just the font size little bigger, it can be without paths, so there is more space


Thanks so much.

**Comment by olefjaerestad** [comment]
I've fixed both points now, @denislutz.

- Point 1 was fixed in https://github.com/olefjaerestad/vscode-circular-dependencies-finder/commit/15087acf9c58b1879b53aac69b70927e3ce1ff1e. Use `cmd+f` (`ctrl+f` on Windows) to search. VS Code doesn't seem to provide an API for displaying the normal "Find" box in a webview (the extension uses a webview to render its contents), so I've used a so-called "Input box" instead. It's a little more intrusive, but it does the job.
- Point 2 was fixed in https://github.com/olefjaerestad/vscode-circular-dependencies-finder/commit/42f4dff966469d253b33956475160cf07598d50d. I was a bit hesitant in removing the filepaths, as a project can easily have multiple files with the same filename, but the filepath is still available as a tooltip when hovering, as well as in the JSON view, so I'm thinking it's worth it.

The fixes are available in version 1.1.0 of the extension. Install it directly from inside VS Code or from https://marketplace.visualstudio.com/items?itemName=olefjaerestad.vscode-circular-dependencies-finder.

Changelog available here: https://marketplace.visualstudio.com/items/olefjaerestad.vscode-circular-dependencies-finder/changelog.
